lib/logstash/outputs/jira.rb

```diff
@@ -83,17 +83,22 @@ def receive(event)
       config.username = @username
       config.password = @password
       config.uri = @host
-      config.auth_type = :cookie
+      config.auth_type = :basic
       config.api_version = "latest"
     end
 
 
+    # Limit issue summary to 255 characters
+    summary = event.sprintf(@summary)
+    summary = "#{summary[0,252]}..." if summary.length > 255
+
 issue = Jiralicious::Issue.new
     issue.fields.set_id("project", @projectid) # would have prefered a project key, https://github.com/jstewart/jiralicious/issues/16
-    issue.fields.set("summary", @summary)
+    issue.fields.set("summary", summary)
+    issue.fields.set("description", event.sprintf(event.to_hash.to_yaml))
```

Comment:
@hudolejev can you add in the plugin documentation an example of output of this field please